docs/v3_services.rst

```diff
@@ -1,89 +1,103 @@
+   from scratch.
+
+How to distinguish between v2 and v3 onion services
+---------------------------------------------------
+
+If your *Source Interface* address is 56 characters long, you are already using v3 
+onion services:
 
 .. code-block:: none
 
   http://vww6ybal4bd7szmgncyruucpgfkqahzddi37ktceo3ah7ngmcopnpyyd.onion/
 
-This makes them easily distinguishable from 16-character v2 onion addresses,
-such as:
+If your *Source Interface* address is 16-characters long, you are still using 
+v2 onion services, and need to transition to v3 in order to keep using SecureDrop:
 
 .. code-block:: none
 
-  http://secrdrop5wyphb5x.onion/
+  http://secrdrop5wyphb5x.onion/  
 
-.. important::
+What an upgrade entails
+-----------------------
 
-   If you are currently advertising a 16-character v2 address like the above
-   to your sources, it will become unreachable in February 2021. You must
-   update to v3 onion services before then.
+Enabling v3 onion services will have the following effects:
 
-   There is no downgrade path from v3 to v2 using the ``securedrop-admin``
-   tool. We recommend that you follow the v2+v3 migration path below, and test v3
-   functionality thoroughly before :ref:`disabling v2 onion services <disable_v2>`.
-
-Migrating from v2 to v3 only or v2+v3
--------------------------------------
-
-Preparing for the migration
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Before starting the migration process, you should decide whether to move
-straight to v3 only, or move to v2+v3 temporarily. As the URLs of your onion
-services will change with the move to v3, moving to v2+v3 first will allow
-you to minimize the impact of the migration on sources and journalists.
-
-URL changes will affect the following:
-
-- The *Source Interface* address will change - once the migration is complete,
-  you will need to update your landing page and other resources that reference
-  the address, such as your SecureDrop directory entry.
-- *Journalist* and *Admin Workstations* will need to be updated to use the v3
-  addresses of the *Journalist* and *Source Interface*, and the SSH proxy
-  services if your instance is using SSH over Tor.
+- You will get a new, 56-character *Source Interface* onion address.
+  If you follow our recommended steps, this will work alongside your existing
+  v2 address. When you're ready to announce the change, you will need to update 
+  your landing page and other resources that reference your *Source Interface* 
+  address, such as your SecureDrop directory entry.
+- *Journalist* and *Admin Workstations* will need to be updated by your 
+  administrator so that they contain the new onion service addresses and
+  credentials.
 - If your instance uses HTTPS, you will need to provision a new certificate for
   the v3 *Source Interface* address - this will need to be done `after` the new
   address has been generated.
 
-.. note:: If your certificate provisioning process requires validation of the
```

Comment:
Again, this is overly simple IMO, we should be upfront with folks about why they need to contact us.

Reply:
I just switched the order of these sentences, nothing was removed. Do you still feel like it's too simplistic? If so I can revert, but I think the note/callout reads clearer this way.